control-plane/tests/test_service_persistence.py

```
@@ -7240,6 +7240,160 @@ def test_service_rejects_isolated_executor_delegation_when_target_scope_drifts(
+            },
+        )
+        self.assertEqual(
+            service.get_record(ActionRequestRecord, action_request.action_request_id),
+            action_request,
+        )
+        self.assertEqual(store.list(ActionExecutionRecord), ())
+
+    def test_service_rejects_reviewed_action_request_creation_when_malformed_or_out_of_scope(
+        self,
+    ) -> None:
+        _, service, promoted_case, evidence_id, reviewed_at = (
```

Ruff (0.15.9)

[warning] Unpacked variable `evidence_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)